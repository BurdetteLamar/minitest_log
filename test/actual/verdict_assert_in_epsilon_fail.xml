<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_in_epsilon?' type='fail'>
    <section_ name='Args=[3, 2, 0, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_in_epsilon?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Integer' value='3'/>
        <actual_ class='Integer' value='2'/>
        <epsilon_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected |3 - 2| (1) to be &lt;= 0.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[4, 3, 0, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_in_epsilon?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Integer' value='4'/>
        <actual_ class='Integer' value='3'/>
        <epsilon_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected |4 - 3| (1) to be &lt;= 0.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
