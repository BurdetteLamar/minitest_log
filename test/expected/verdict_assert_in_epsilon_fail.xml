<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_in_epsilon?' type='fail'>
    <section name='Args=[3, 2, 0]'>
      <verdict method='verdict_assert_in_epsilon?' outcome='failed' id='0' message='Message 0'>
        <exp_value>
          3
        </exp_value>
        <act_value>
          2
        </act_value>
        <epsilon>
          0
        </epsilon>
        <exception class='Minitest::Assertion'>
          <message>
            Expected |3 - 2| (1) to be &lt;= 0.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[4, 3, 0]'>
      <verdict method='verdict_assert_in_epsilon?' outcome='failed' id='1' message='Message 1'>
        <exp_value>
          4
        </exp_value>
        <act_value>
          3
        </act_value>
        <epsilon>
          0
        </epsilon>
        <exception class='Minitest::Assertion'>
          <message>
            Expected |4 - 3| (1) to be &lt;= 0.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
